src/Sdk/NewsroomSubscriptions/types.ts

```diff
@@ -0,0 +1,38 @@
+import { Contact, UploadcareImage } from '../../types';
+
+export interface NewsroomSubscriptionCreateRequest {
+    email_address: string;
+    locale?: string;
+    url?: string | null;
+    ip_address?: string | null;
+    visitor_uid?: string | null;
+    session_uid?: string | null;
+    contact?: {
+        contact_type: Contact['contact_type'];
+        company_name: Contact['company_name'];
```

Comment:
`company_name` is only required when `contact_type === 'organisation'`

```suggestion
        company_name?: Contact['company_name'];
```